src/net/sourceforge/kolmafia/StaticEntity.java

```diff
@@ -81,17 +81,33 @@ public static final String getVersion() {
     if (StaticEntity.cachedVersionName == null) {
       StringBuilder versionName =
           new StringBuilder(PRODUCT_NAME).append(" r").append(StaticEntity.getRevision());
+
+      if (isCodeModified()) {
+
```

Comment:
Extraneous newline?

Reply:
yep!  probably because I hacked the function up between commits.   will clean up.  Spotless didn't find that, btw.